CHANGELOG.md

```
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Agent memory compaction now runs by default for task, project, and day
   agents, and the Settings flag that controlled it has been removed.
 
+### Changed
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)